src/components/interaction-designer/clipboard/ClipboardRoot.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div class="clipboard-root-view">
+    <header class="d-flex justify-content-between">
+      <h3>
+        {{ 'flow-builder.clipboard-simulator' | trans }}
+      </h3>
+      <i class="glyphicon glyphicon-remove cursor-pointer align-self-center h4"
+         @click="closeSimulator"></i>
+    </header>
+    <main>
+      <div v-for="(blockPrompt, i) in blockPrompts" :key="i" class="mt-2">
+        <component :is="blockPrompt.config.kind"
+                   :prompt="blockPrompt"
+                   :context="context"
+                   :go-next="goNext">
+        </component>
+      </div>
+      <div v-if="unsupportedBlockName" class="mt-2">
+        <unsupported-block :block-name="unsupportedBlockName"  />
+      </div>
+    </main>
+    <footer v-if="isComplete" class="mt-2">
+      <div class="card">
+        <div class="card-body">
+          <h5 class="card-title">{{'flow-builder.completed' | trans}}!</h5>
+        </div>
+      </div>
+    </footer>
+  </div>
+</template>
+
+<script lang="ts">
+import lang from '@/lib/filters/lang'
+import { mapActions, mapGetters } from 'vuex'
+import {
+  IContext,
+  createContextDataObjectFor,
+  FlowRunner,
+  IRichCursorInputRequired,
+  SupportedMode,
+  IContact,
+} from '@floip/flow-runner'
+import Message from './prompt-kinds/Message.vue'
+import Numeric from './prompt-kinds/Numeric.vue'
+import Open from './prompt-kinds/Open.vue'
+import SelectOne from './prompt-kinds/SelectOne.vue'
+import SelectMany from './prompt-kinds/SelectMany.vue'
+import UnsupportedBlock from './shared/UnsupportedBlock.vue'
+
+export default {
+  components: {
+    Message,
+    Numeric,
+    Open,
+    SelectOne,
+    SelectMany,
+    UnsupportedBlock,
+  },
+  mixins: [lang],
+  data() {
+    return {
+      runner: FlowRunner,
+      blockPrompts: [],
+      context: {},
+      isComplete: false,
+      unsupportedBlockName: '',
+    }
+  },
+  created() {
+    this.initializeFlowRunner()
+  },
+  methods: {
+    ...mapGetters('flow', ['getFlowsState']),
```

Comment:
@poojahpatil90 , I think getters should be under `computed` not `methods`. That way it will be its value will be updated by vue. Then you can call it directly with `this.getFlowsState` not `this.getFlowsState()`

Reply:
In addition, I'd suggest we not use action oriented getter names, since at the call site they're referenced as variables/state.